src/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To Do List</title>
+  </head>
+  <body>
+    <main id="todo-wrapper">
+      <div class="todo-header">
+        <h4>Title</h4>
```

Comment:
- [ ] (OPTIONAL) Consider replacing `Title` here with `Today's To Do` as it is more explicit and it also follows the design 👍